features/login.feature

```diff
@@ -0,0 +1,29 @@
+Feature: New customer Registration
+
+
+Background: User is Logged In
+	Given the user navigates to Homepage
+	When the user clicks on the create a account button
+	Then the user should be navigated to customer account page
+
+Scenario: Empty registration Authentication (Negative scenario)
```

Comment:
@karthikeyan7010 There will be n number of scenarios in negative cases compared to positive case.